level-2/N개의-최소공배수.py

```diff
@@ -0,0 +1,18 @@
+def gcd(a, b):
+    if a > b:
+        a, b = b, a
+    while b%a != 0:
+        b, a = a, b%a
```

Comment:
Some comments about this part would be great :)

Reply:
Comment on that part is added.